gg/meson.build

```diff
@@ -25,6 +25,7 @@ gg_prpl = shared_library('gg',
   dependencies: [libgadu_dep, libpurple_dep, glib_dep, ws2_32_dep],
   install: true,
   install_dir: plugin_dir,
+  name_suffix: plugin_suffix,
```

Comment:
A minor quibble, but I put `install`/`install_dir` last so that the arguments correspond to the stages, i.e., source/build/install.